Assets/HoloToolkit/Input/Scripts/Focus/FocusManager.cs

```diff
@@ -629,7 +629,8 @@ private void RaycastUnityUI(PointerData pointer, LayerMask[] prioritizedLayerMas
             }
 
             // Check if we need to overwrite the physics raycast info
-            if ((pointer.End.Object == null || overridePhysicsRaycast) && uiRaycastResult.isValid && uiRaycastResult.module.eventCamera == UIRaycastCamera)
+            if ((pointer.End.Object == null || overridePhysicsRaycast) && uiRaycastResult.isValid && 
+			     uiRaycastResult.module != null && uiRaycastResult.module.eventCamera == UIRaycastCamera)
```

Comment:
huh, I thought I added this before.  Must have gotten dropped someplace.

Reply:
So I am adding it for you again ;)